target/tests/tests_views.py

```diff
@@ -56,6 +57,22 @@ def test_create_target(self):
         self.assertEqual(response.status_code, status.HTTP_201_CREATED)
         self.assertEqual(response.data.get('properties').get('user'), self.test_active_user.id)
 
+    def test_create_more_targets_than_it_is_allowed(self):
+        initial_num_of_targets = Target.objects.filter(user=self.test_active_user).count()
+        for i in range(0, (settings.MAX_NUMBER_OF_TARGETS-initial_num_of_targets)):
```

Comment:
Add spaces around the minus sign

Reply:
How flake8 didn't fail in this case. I totally agree.